static/app/views/alerts/wizard/options.tsx

```diff
@@ -19,7 +19,15 @@ export type AlertType =
   | 'cls'
   | 'custom'
   | 'crash_free_sessions'
-  | 'crash_free_users';
+  | 'crash_free_users'
+  // Mobile vitals
+  | 'app_start_cold'
+  | 'app_start_warm'
+  | 'frames_slow'
+  | 'frames_frozen'
+  | 'stall_longest_time'
+  | 'stall_total_time'
+  | 'stall_count';
```

Comment:
@wedamija Taylan (@taylangocmen ) mentioned there may need to be a backend change to add support for these measurements. Can you confirm?